pylint/checkers/base.py

```diff
@@ -1329,6 +1339,17 @@ def _check_name(self, node_type, name, node, confidence=interfaces.HIGH):
         if match is None:
             self._raise_name_warning(node, node_type, name, confidence)
 
+    @staticmethod
+    def _name_became_keyword_in_version(name):
```

Comment:
just a tiny nitpick. Technically, it we'll check if the name will become a keyword in the future. :-) In case we are running pylint with the said version, the program will result in a syntax error anyway.